README.md

```diff
@@ -5,7 +5,14 @@
 It runs with [Prow CI](https://github.com/kubernetes/test-infra/blob/master/prow/) on a [Google Kubernetes Engine Cluster](https://cloud.google.com/kubernetes-engine/).
 It is designed to support adding more k8s providers.
 
-## Run tests manually
+## Overview of the manifest files
+The `/manifest` directory contains all the kubernetes manifest files.
+- `cluster.yaml` : This is used to create the GKE cluster.
```

Comment:
I think we can move this file inside `cluster-infra/` to simplify the Readme and deployment even more.

Reply:
can you move the cluster.yaml inside the cluste-infra folder?

Reply:
oh sorry i missed this one. updated.